store/sqlstore/plugin_store.go

```diff
@@ -92,3 +92,14 @@ func (ps SqlPluginStore) Delete(pluginId, key string) store.StoreChannel {
 		}
 	})
 }
+
+func (ps SqlPluginStore) List(pluginId string) store.StoreChannel {
+	return store.Do(func(result *store.StoreResult) {
+		var keys []string
+		if _, err := ps.GetReplica().Select(&keys, "SELECT PKey FROM PluginKeyValueStore WHERE PluginId = :PluginId", map[string]interface{}{"PluginId": pluginId}); err != nil {
```

Comment:
We're starting to write things to follow more idiomatic go. Can you structure this as:
```go
_, err := ps.GetReplica().Select(...)
if err != nil {
    result.Err = ...
    return
}

result.Data = keys
```